### DISCUSSION TITLE
Is there a way to add a spellchecker to Minder?

### DISCUSSION THREAD

**Discussion by AnotherPersonOnGithub** [phase1geo/Minder#611] [Ideas]
I know it seems silly but a spell check would be nice to have. Is there a way to add one?

**Comment by phase1geo** [comment]
There isn't a way to add a spell checker from the user's perspective currently.

**Reply by AnotherPersonOnGithub** [reply]
Thanks for the quick reply, is this something we can look forward to in the future?

**Reply by phase1geo** [reply]
That would be doable.
